portfolio/src/main/webapp/css/semantic.min.css

```diff
@@ -0,0 +1,372 @@
+ /*
```

Comment:
Why copy the entire file? I suggest use a CDN based import for all libraries you are using. Code duplication is something that is pointless and unnecessary

Reply:
Somehow, I was under the assumption that serving this way was better. I'll fix that.